### PR TITLE
Embed iFrame instead of screenshot for App output in Docs

### PR DESCRIPTION
**Pull request by shahinrostami**
Thank you so much for contributing to Datapane, please help us by filing out this PR template and ensure you have read the [CONTRIBUTING.md](../CONTRIBUTING.md).

It may take a few days to review your PR and merge it - please bear with us!

Thanks!

fixes #

## Prerequsites

- [x] Has been tested locally
- [ ] If a bugfix, have included a breaking test if possible

## Proposed Changes

Embed iFrame instead of screenshot for App output in Docs

### CODE REVIEW

File: docs/dpdocsutils/dpdocsutils/previews.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def app_to_image(app_path, image_path, width, height):
     )
 
 
-def embed_local_app(app_path, width, height, iframe=False):
+def embed_local_app(app_path, width, height, iframe=True):
     if iframe:
         return IFrame(app_path, width=width, height=height)
```
Comment (antdking) [inline docs/dpdocsutils/dpdocsutils/previews.py:41]: I wonder if we can set `loading="lazy"` here to defer fetching..

Comment (antdking) [reply docs/dpdocsutils/dpdocsutils/previews.py:41]: Running a build locally, you add it in using `extras=['loading="lazy"'].

```suggestion
        return IFrame(app_path, width=width, height=height, extras=['loading="lazy"'])
```

Comment (antdking) [reply docs/dpdocsutils/dpdocsutils/previews.py:41]: Works a treat on Chrome, not supported on Firefox.

I think we want this, @shahinrostami.
Have a play with scrolling through the Accessing Data in Python guide. Causes more caching to happen, too (requests aren't racing each other anymore).

Comment (shahinrostami) [reply docs/dpdocsutils/dpdocsutils/previews.py]: Nice, have commit the suggestion. It didn't work in Safari either, unfortunately.

**Review by antdking — APPROVED** [review]
Looking good, nice to have the interactive parts brought closer into the docs

**Pull request merged**